web/src/lib/components/shared-components/navigation-bar/navigation-bar.svelte

```diff
@@ -76,33 +76,38 @@
         {/if}
 
         {#if user.isAdmin}
-          <a data-sveltekit-preload-data="hover" href={AppRoute.ADMIN_USER_MANAGEMENT}>
-            <div class="hidden sm:block">
-              <LinkButton>
-                <span
+          <a data-sveltekit-preload-data="hover" href={AppRoute.ADMIN_USER_MANAGEMENT}
+            aria-label="Administration"
+            aria-current={$page.url.pathname.includes('/admin')
+            ? 'page'
+            : null
+            }
+            >
+            <div class="inline-flex items-center justify-center transition-colors dark:text-immich-dark-fg p-2 font-medium rounded-lg">
+              <div class="hidden sm:block">
+                  <span
+                    class={$page.url.pathname.includes('/admin')
+                      ? 'item text-immich-primary underline dark:text-immich-dark-primary'
+                      : ''}
+                  >
+                    Administration
+                  </span>
+              </div>
+              <div class="block sm:hidden" aria-hidden="true">
+                  <Cog
+                    size="1.5em"
+                    class="dark:text-immich-dark-fg {$page.url.pathname.includes('/admin')
+                      ? 'text-immich-primary dark:text-immich-dark-primary'
+                      : ''}"
+                  />
+                <hr
+                  role="decoration"
```

Comment:
The linting checks are complaining that a hr cannot have a role. Can you fix/address this please?

Reply:
Yes, I have a better solution on it's way.